app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,19 @@
+class SessionsController < ApplicationController
+  def new; end
+
+  def create
+    user = User.find_by(email: params[:session][:email].downcase)
+    if user&.authenticate(params[:session][:password])
+      log_in user
+      redirect_to :root
+    else
+      flash.now[:danger] = t ("invalid")
```

Comment:
```suggestion
      flash.now[:danger] = t "invalid"
```

đặt lại tên biến I18n em ơi, invalid là invalid cái j chứ, clear tí nào

Reply:
da.